mifosng-android/src/release/java/com/mifos/api/local/MifosSQLCipherHelper.java

```diff
@@ -0,0 +1,18 @@
+package com.mifos.api.local;
+
+import com.raizlabs.android.dbflow.config.DatabaseDefinition;
+import com.raizlabs.android.dbflow.structure.database.DatabaseHelperListener;
+import com.raizlabs.dbflow.android.sqlcipher.SQLCipherOpenHelper;
+
+public class MifosSQLCipherHelper extends SQLCipherOpenHelper {
+
+    public MifosSQLCipherHelper(DatabaseDefinition databaseDefinition,
+                                DatabaseHelperListener listener) {
+        super(databaseDefinition, listener);
+    }
+
+    @Override
+    protected String getCipherSecret() {
+        return "dbflow-rules";
```

Comment:
I have two way to implemented.
1. What about if we make the MD5 hash of username and password entered by the user. And save them in the custom data table on the server.
2. Add an extra screen and give the user to enter the database encryption key after the login and save then on server and double encrypt the key and save in Shared Preference So no one can get the key other than admin.